MobileWalletTests/TariLibWrapperTests.swift

```diff
@@ -388,7 +388,8 @@ class TariLibWrapperTests: XCTestCase {
                     databaseName: self.dbName,
                     publicAddress: address,
                     discoveryTimeoutSec: TariSettings.shared.discoveryTimeoutSec,
-                    safMessageDurationSec: TariSettings.shared.safMessageDurationSec
+                    safMessageDurationSec: TariSettings.shared.safMessageDurationSec,
+                    networkName: "weatherwax"
```

Comment:
We keep network names in `TariNetwork` enum. Please consider this suggestion:
```suggestion
               networkName: TariNetwork.weatherwax.rawValue
```